packages/components/radio-group/src/utils/helper-functions.js

```diff
@@ -0,0 +1,36 @@
+import { constants } from '@tradeshift/elements';
+import { directionTypes } from './constants';
+
+const { keyboardEventKeys } = constants;
+
+export function getDirectionByKey(keyCode) {
+	switch (keyCode) {
+		case keyboardEventKeys.ARROW_RIGHT:
+		case keyboardEventKeys.ARROW_DOWN:
+			return directionTypes.FORWARD;
+		case keyboardEventKeys.ARROW_LEFT:
+		case keyboardEventKeys.ARROW_UP:
+			return directionTypes.BACKWARD;
+		default:
+			return directionTypes.NONE;
+	}
+}
+
+export function getNextElementIndex(direction, index, collectionLength) {
+	switch (direction) {
+		case directionTypes.FORWARD:
+			if (index === collectionLength - 1) {
+				return 0;
+			}
+			return ++index;
+		case directionTypes.BACKWARD:
+			if (index === 0) {
+				return collectionLength - 1;
+			}
+			return --index;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Extract this decrement operation into a dedicated statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=javascript%3AS881)